diff --git a/dist/error-d5becc6e.d.ts b/dist/error-d5becc6e.d.ts
index 1629b23b4d2b375d02b6d6ebda2e722eb4d934c2..cab259b401bfd6a7d747bd66f143c572d12fdd60 100644
--- a/dist/error-d5becc6e.d.ts
+++ b/dist/error-d5becc6e.d.ts
@@ -1,7 +1,7 @@
-type Fetch = typeof globalThis.fetch;
-type RequestInfo = globalThis.RequestInfo;
-type RequestInit = globalThis.RequestInit;
-type Response = globalThis.Response;
+type Fetch = typeof import("node-fetch");
+type RequestInfo = import("node-fetch").RequestInfo;
+type RequestInit = import("node-fetch").RequestInit;
+type Response = import("node-fetch").Response;
 
 interface ResponseMap {
     blob: Blob;